cmd/jujud-controller/agent/model/manifolds.go

```diff
@@ -351,12 +350,14 @@ func IAASManifolds(config ManifoldsConfig) dependency.Manifolds {
 	controllerTag := agentConfig.Controller()
 	modelTag := agentConfig.Model()
 	manifolds := dependency.Manifolds{
-		providerTrackerName: ifCredentialValid(ifResponsible(providertracker.Manifold(providertracker.ManifoldConfig{
+		providerTrackerName: ifCredentialValid(ifResponsible(providertracker.Manifold(providertracker.ManifoldConfig[environs.Environ]{
```

Comment:
I haven't checked specifics yet, but if we remove reliance on the API for model workers, it may turn out that we need this whether or not we are the singular model worker engine.